rhizome/host/bin/setup-vm

```diff
@@ -51,6 +51,7 @@ begin
   swap_size_bytes = params["swap_size_bytes"]
   pci_devices = params.fetch("pci_devices", [])
   boot_image = params["boot_image"]
+  readonly_image = params.fetch("readonly_image", "")
```

Comment:
this is a worthy feature all on its own. the patch is not *so* large that it's a problem to inline it (but it would be even better to herald this enhancement in its own commit message), but support for read only volumes is definitely its own cool and good thing.

Reply:
added separate commit.